exporter/awsemfexporter/emf_exporter_test.go

```diff
@@ -24,11 +24,6 @@ import (
 
 const defaultRetryCount = 1
 
-func init() {
-	os.Setenv("AWS_ACCESS_KEY_ID", "test")
-	os.Setenv("AWS_SECRET_ACCESS_KEY", "test")
-}
-
```

Comment:
These weren't required anymore and were causing warnings for unchecked error. Opted to remove dead code.